cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutorState.java

```diff
@@ -119,11 +177,14 @@ public int numFinishedPartitionMovements() {
         execState.put("state", _state);
         break;
       case REPLICA_MOVEMENT_TASK_IN_PROGRESS:
+      case STOPPING:
         execState.put("state", _state);
         execState.put("numTotalPartitions", numTotalPartitionMovements());
         execState.put("totalDataToMove", totalDataToMoveInMB());
         execState.put("numFinishedPartitions", _numFinishedPartitionMovements);
         execState.put("finishedDataMovement", _finishedDataMovementInMB);
+        execState.put("abortingPartitions", _abortingPartitionMovements);
+        execState.put("abortedPartitions", _abortedPartitionMovements);
```

Comment:
Why does `getJsonStructure` and `toString` return results of different fields for `STOPPING`? -- i.e. abortingPartitions and abortingPartitions in getJsonStructure vs. abortedPartitionMovements and deadPartitionMovements in toString.